akka-projection-core/src/main/scala/akka/projection/internal/spi/ProjectionMetrics.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.internal.spi
+
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.InternalStableApi
+import akka.projection.ProjectionId
+
+/**
+ * INTERNAL API
+ */
+@InternalStableApi private[akka] trait ProjectionMetrics {
```

Comment:
Having some hooks for a projection worker life cycle could be useful as well. It could provide some additional metrics such as number of active projections / workers.

Also it would be useful to provide an ActorRef or something else that represents a projection worker. It could be useful for attaching some metadata like instruments and synchronize its creation deletion with the projection worker life cycle.